streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -70,13 +99,12 @@ public void shouldAllowJoinUnmaterializedFilteredKTable() {
         final KTable<Bytes, String> filteredKTable = builder.<Bytes, String>table("table-topic").filter(MockPredicate.<Bytes, String>allGoodPredicate());
         builder.<Bytes, String>stream("stream-topic").join(filteredKTable, MockValueJoiner.TOSTRING_JOINER);
 
-        driver.setUp(builder, TestUtils.tempDirectory());
-
-        ProcessorTopology topology = builder.internalTopologyBuilder.build();
```

Comment:
We can keep this line and also don't need changes below. Why did you move this to `TopologyTestDriver`? Seems to be an unnecessary redirection?